vars/ploigosWorkflowStandard.groovy

```diff
@@ -243,6 +250,33 @@ def call(Map paramsMap) {
     /* Name of the virtual environment to set up in the given home worksapce. */
     String WORKFLOW_WORKER_VENV_NAME = 'venv-ploigos'
 
+    /* Directory into which platform configuration is mounted, if applicable */
+    String PLATFORM_CONFIG_DIR = "/opt/platform-config"
+
+    /* Additional mounts for agent containers, if separatePlatformConfig == true */
+    String PLATFORM_MOUNTS = params.separatePlatformConfig ? """
```

Comment:
I think this might be a bit long for a ternary operator. Any time you have new lines I think it kills the readability. My 2 cents